go-controller/cmd/ovn-k8s-cni-overlay/app/helper_windows.go

```diff
@@ -194,6 +194,13 @@ func ConfigureInterface(args *skel.CmdArgs, namespace string, podName string, ma
 		return nil, fmt.Errorf("failure in plugging pod interface: %v  %q", err, string(out))
 	}
 
+	if err = clearPodBandwidth(args.ContainerID); err != nil {
```

Comment:
I would say to comment out those functions and just leave them there. Add a TODO as well for them please, we will take care of it.

Also as @aserdean suggested, put a log message saying that QoS is not supported yet on Windows.